### ISSUE TITLE
[IntelliJ-ThirdParty][A11yMAS][Accessibility][JAWS/NVDA][MAS4.2.1][IntelliJ][Wipro]: NVDA and Jaws are not announcing the content present in " Browse Repositories" dialog sequentially

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1522]
Repro Steps:

1. Open and Start IntelliJ IDEA.
2. Navigate and select Settings Menu item under File menu.
3. In Settings dialog  box, navigate and select Plugins, and then Select Browse repositories  button.
4. In Browse Repositories window , Press "Insert+B" to read  the entire dialog then observe the NVDA and JAWS Behavior

**Expected Result:**
NVDA and JAWS Should announce in the following manner
- Browse Repositories  dialog  
- Search edit field
search button
- Refresh button
- Category All button
list items
- Sort by: name
- table
All plugins
TEAMWORK
 AccuRev Integration for IntelliJ
27 downloads
Updated 1/31/2018
v2017
- Manage repositories...  button  alt+M
- HTTP Proxy Settings...  button  alt+H
- Reload List of Plugins  button  Reload List of Plugins
- close button

**Actual Result:**
NVDA is announcing as
Browse Repositories  dialog  Sort by: name
TEAMWORK
AccuRev Integration for IntelliJ
27 downloads
Updated 1/31/2018
v2017
Manage repositories...  button  alt+M
HTTP Proxy Settings...  button  alt+H
Reload List of Plugins  button  Reload List of Plugins
Category: All  button
edit
blank
table
VCS INTEGRATION
.ignore  
3 weeks ago
7,174,454
**Jaws behavior:**
Browse Repositories dialog
Sort by: name VCS INTEGRATION .ignore7174454 downloads Updated 11-01-2018 v2.4.0 
Close Button

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jet brains.
OSBuild Version: Version 1709(Os Build 16299.192)

![browse repositories-insert-b](https://user-images.githubusercontent.com/22956130/36189004-d26d7606-1175-11e8-9587-fc6f5e697431.jpg)
![browse repositories hierarcy](https://user-images.githubusercontent.com/22956130/36189077-2a2ec3ea-1176-11e8-995f-5230a5323490.jpg)

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by Eskibear** [comment]
And it's out of the tookit's scope.

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749973
